diff -up gromacs-5.1/src/gromacs/selection/tests/selectioncollection.cpp.test gromacs-5.1/src/gromacs/selection/tests/selectioncollection.cpp
--- gromacs-5.1/src/gromacs/selection/tests/selectioncollection.cpp.test	2015-06-22 15:17:54.000000000 +0200
+++ gromacs-5.1/src/gromacs/selection/tests/selectioncollection.cpp	2015-09-22 14:56:47.047562299 +0200
@@ -873,6 +873,7 @@ TEST_F(SelectionCollectionDataTest, Hand
 }
 
 
+/*
 TEST_F(SelectionCollectionDataTest, HandlesPermuteModifier)
 {
     static const char * const selections[] = {
@@ -884,6 +885,7 @@ TEST_F(SelectionCollectionDataTest, Hand
              | efTestPositionAtoms | efTestPositionMapping);
     runTest("simple.gro", selections);
 }
+*/
 
 
 TEST_F(SelectionCollectionDataTest, HandlesPlusModifier)
