diff -up gromacs-bec9c8757e59cae58fc61ed841c0bb73c84079db/docs/conf.py.sphinx-no-man gromacs-bec9c8757e59cae58fc61ed841c0bb73c84079db/docs/conf.py
--- gromacs-bec9c8757e59cae58fc61ed841c0bb73c84079db/docs/conf.py.sphinx-no-man	2016-03-18 16:05:26.000000000 +0100
+++ gromacs-bec9c8757e59cae58fc61ed841c0bb73c84079db/docs/conf.py	2016-03-24 10:26:42.361145156 +0100
@@ -302,7 +302,7 @@ latex_documents = [
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-execfile('conf-man.py')
+#execfile('conf-man.py')
 
 # If true, show URL addresses after external links.
 #man_show_urls = False
