diff -up gromacs-5.1/src/programs/mdrun/tests/moduletest.cpp.opencl gromacs-5.1/src/programs/mdrun/tests/moduletest.cpp
--- gromacs-5.1/src/programs/mdrun/tests/moduletest.cpp.opencl	2015-06-22 15:17:54.000000000 +0200
+++ gromacs-5.1/src/programs/mdrun/tests/moduletest.cpp	2015-09-23 23:01:35.456564790 +0200
@@ -220,7 +220,7 @@ SimulationRunner::callMdrun(const Comman
 #ifdef GMX_MPI
 #  ifdef GMX_GPU
 #    ifdef GMX_THREAD_MPI
-    int         numGpusNeeded = g_numThreads;
+    int         numGpusNeeded = 1;
 #    else   /* Must be real MPI */
     int         numGpusNeeded = gmx_node_num();
 #    endif
