diff -up gromacs-5.0.4/src/gromacs/trajectoryanalysis/tests/angle.cpp.issue1716 gromacs-5.0.4/src/gromacs/trajectoryanalysis/tests/angle.cpp
--- gromacs-5.0.4/src/gromacs/trajectoryanalysis/tests/angle.cpp.issue1716	2014-07-07 15:03:16.000000000 +0200
+++ gromacs-5.0.4/src/gromacs/trajectoryanalysis/tests/angle.cpp	2015-04-23 13:47:59.539750157 +0200
@@ -184,21 +184,4 @@ TEST_F(AngleModuleTest, HandlesOneVsMult
     runTest(CommandLine(cmdline));
 }
 
-TEST_F(AngleModuleTest, HandlesOneVsMultipleVectorGroupsAngles)
-{
-    const char *const cmdline[] = {
-        "angle",
-        "-g1", "vector",
-        "-group1",
-        "resname RV1 and name A1 A2",
-        "resname RV3 RV4 and name A1 A2",
-        "-g2", "plane",
-        "-group2",
-        "resname RP1 RP2 and name A1 A2 A3",
-        "-binw", "60"
-    };
-    setTopology("angle.gro");
-    runTest(CommandLine(cmdline));
-}
-
 } // namespace
