diff -up gromacs-2016/src/gromacs/CMakeLists.txt.tng gromacs-2016/src/gromacs/CMakeLists.txt
--- gromacs-2016/src/gromacs/CMakeLists.txt.tng	2016-07-09 02:55:38.000000000 +0200
+++ gromacs-2016/src/gromacs/CMakeLists.txt	2016-08-23 23:41:58.359326148 +0200
@@ -122,7 +122,6 @@ tmpi_get_source_list(THREAD_MPI_SOURCES
 list(APPEND LIBGROMACS_SOURCES ${THREAD_MPI_SOURCES})
 
 list(APPEND LIBGROMACS_SOURCES ${TNG_SOURCES})
-tng_set_source_properties(WITH_ZLIB ${HAVE_ZLIB})
 
 get_lmfit_properties(LMFIT_SOURCES LMFIT_LIBRARIES LMFIT_INCLUDE_DIRECTORY LMFIT_INCLUDE_DIR_ORDER)
 include_directories(${LMFIT_INCLUDE_DIR_ORDER} SYSTEM "${LMFIT_INCLUDE_DIRECTORY}")
