diff -up gromacs-2016/src/gromacs/CMakeLists.txt.lmfit gromacs-2016/src/gromacs/CMakeLists.txt
--- gromacs-2016/src/gromacs/CMakeLists.txt.lmfit	2016-07-09 02:55:38.000000000 +0200
+++ gromacs-2016/src/gromacs/CMakeLists.txt	2016-08-24 00:20:34.262790989 +0200
@@ -124,7 +124,7 @@ list(APPEND LIBGROMACS_SOURCES ${THREAD_
 list(APPEND LIBGROMACS_SOURCES ${TNG_SOURCES})
 tng_set_source_properties(WITH_ZLIB ${HAVE_ZLIB})
 
-get_lmfit_properties(LMFIT_SOURCES LMFIT_LIBRARIES_TO_LINK LMFIT_INCLUDE_DIRECTORY LMFIT_INCLUDE_DIR_ORDER)
+get_lmfit_properties(LMFIT_SOURCES LMFIT_LIBRARIES LMFIT_INCLUDE_DIRECTORY LMFIT_INCLUDE_DIR_ORDER)
 include_directories(${LMFIT_INCLUDE_DIR_ORDER} SYSTEM "${LMFIT_INCLUDE_DIRECTORY}")
 list(APPEND LIBGROMACS_SOURCES ${LMFIT_SOURCES})
 
@@ -183,7 +183,7 @@ target_link_libraries(libgromacs
                       ${TNG_IO_LIBRARIES}
                       ${FFT_LIBRARIES} ${LINEAR_ALGEBRA_LIBRARIES}
                       ${XML_LIBRARIES}
-                      ${LMFIT_LIBRARIES_TO_LINK}
+                      ${LMFIT_LIBRARIES}
                       ${THREAD_LIB} ${GMX_SHARED_LINKER_FLAGS} ${OPENCL_LIBRARIES}
                       ${GMX_STDLIB_LIBRARIES})
 set_target_properties(libgromacs PROPERTIES
