diff --git a/src/gromacs/gmxana/gmx_do_dssp.cpp b/src/gromacs/gmxana/gmx_do_dssp.cpp
index a8aab1bc2..184592b7e 100644
--- a/src/gromacs/gmxana/gmx_do_dssp.cpp
+++ b/src/gromacs/gmxana/gmx_do_dssp.cpp
@@ -445,7 +445,7 @@ int gmx_do_dssp(int argc, char *argv[])
         "calling the dssp program. If you do not have the dssp program,",
         "get it from http://swift.cmbi.ru.nl/gv/dssp. [THISMODULE] assumes ",
         "that the dssp executable is located in ",
-        "[TT]/usr/local/bin/dssp[tt]. If this is not the case, then you should",
+        "[TT]/usr/bin/mkdssp[tt]. If this is not the case, then you should",
         "set an environment variable [TT]DSSP[tt] pointing to the dssp",
         "executable, e.g.: [PAR]",
         "[TT]setenv DSSP /opt/dssp/bin/dssp[tt][PAR]",
@@ -590,7 +590,7 @@ int gmx_do_dssp(int argc, char *argv[])
 
     if ((dptr = getenv("DSSP")) == nullptr)
     {
-        dptr = "/usr/local/bin/dssp";
+        dptr = "/usr/bin/mkdssp";
     }
     if (!gmx_fexist(dptr))
     {
