diff --git a/src/gromacs/hardware/tests/hardwaretopology.cpp b/src/gromacs/hardware/tests/hardwaretopology.cpp
index ed7897b01..c31d4b9cd 100644
--- a/src/gromacs/hardware/tests/hardwaretopology.cpp
+++ b/src/gromacs/hardware/tests/hardwaretopology.cpp
@@ -185,12 +185,14 @@ TEST(HardwareTopologyTest, NumaCacheSelfconsistency)
             }
         }
 
+#ifndef __aarch64__
         // Check cache. The hwloc cache detection is fragile and can report
         // 0 for line size or associativity (=unknown), so we just check the size.
         for (auto &c : hwTop.machine().caches)
         {
             EXPECT_GT(c.size, 0);
         }
+#endif
     }
 }
 
