--- gromacs-4.0_rc2/scripts/GMXRC.csh.in.orig	2001-08-18 14:07:31.000000000 +0300
+++ gromacs-4.0_rc2/scripts/GMXRC.csh.in	2008-09-28 15:33:26.000000000 +0300
@@ -1,3 +1,4 @@
+#!/bin/csh
 # csh/tcsh configuration file for Gromacs.
 # First we remove previous Gromacs stuff from paths 
 # by selecting everything else. 
--- gromacs-4.0_rc2/scripts/GMXRC.bash.in.orig	2001-08-18 14:07:31.000000000 +0300
+++ gromacs-4.0_rc2/scripts/GMXRC.bash.in	2008-09-28 15:33:08.000000000 +0300
@@ -1,3 +1,4 @@
+#!/bin/bash
 # zsh configuration file for Gromacs
 # First we remove old gromacs stuff from the paths
 # by selecting everything else.
--- gromacs-4.0_rc2/scripts/GMXRC.in.orig	2001-08-18 14:07:31.000000000 +0300
+++ gromacs-4.0_rc2/scripts/GMXRC.in	2008-09-28 15:33:44.000000000 +0300
@@ -1,3 +1,4 @@
+#!/bin/sh
 # This is a convenience script to determine which
 # type of shell you have, and then run GMXRC.[csh|bash|zsh]
 # from the Gromacs binary directory.
--- gromacs-4.0_rc2/scripts/GMXRC.zsh.in.orig	2001-08-18 14:07:31.000000000 +0300
+++ gromacs-4.0_rc2/scripts/GMXRC.zsh.in	2008-09-28 15:33:34.000000000 +0300
@@ -1,3 +1,4 @@
+#!/bin/zsh
 # zsh configuration file for Gromacs
 # First we remove old gromacs stuff from the paths
 # by selecting everything else.
